drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -221,7 +221,6 @@ def items_without_answers():
             "target_img_description": self.target_img_description,
             "item_background_color": self.item_background_color or None,
             "item_text_color": self.item_text_color or None,
-            "initial_feedback": self.data['feedback']['start'],
```

Comment:
@cahrens no, it is still shown in feedback area - it is passed through `get_user_state` handler, rather than here.